static/app/components/metrics/mriSelect/index.tsx

```diff
@@ -214,6 +216,14 @@ export const MRISelect = memo(function MRISelect({
     [onChange]
   );
 
+  const maxLength = useMemo(() => {
```

Comment:
https://github.com/user-attachments/assets/cef3f4de-ba9c-425e-b66b-c5dae722ab1e

I am afraid we still have to tweak this function a little more

Reply:
This is hard to achieve with a single number because small devices have different viewport sizes. Ideally, CSS should handle this. However, since we treat the string differently based on its format, we need a different approach 🤔

Reply:
Added new breakpoint to deal with this use case